thefuck/rules/git_commit_amend.py

```diff
@@ -0,0 +1,11 @@
+from thefuck.specific.git import git_support
+
+
+@git_support
+def match(command):
+    return ('commit' in command.script_parts)
```

Comment:
It'd be nice to only offer this if the commit was successful, but it's not obvious what a 'successful' commit may or may not output.